web/libs/editor/src/tags/object/AudioUltra/model.js

```diff
@@ -131,6 +132,9 @@ export const AudioModel = types.compose(
     })
     .volatile(() => ({
       errors: [],
+      stageRef: createRef(),
+      _ws: null,
+      _wf_frame: null,
```

Comment:
```suggestion
      _frame: null,
```

Reply:
I know there was historically a `_ws_region` which likely influenced this naming, but that was a mistake it should have been camelCase not snake_case. I don't know how ingrained `_ws_region` is already, but if at all possible could we rename both of these to camelCase like `_wfFrame` and `_wsRegion`.

Reply:
@bmartel I've returned to the `_ws_region` as there are a lot of places which it affects. In general, the worst part is that we have three versions of the `Audio` tag and if we change it here we will need to change it everywhere. I prefer not to touch it until we remove the related feature flags and legacy versions.